tests/robot-tests/tests/general_public/methodologies_page.robot

```diff
@@ -1,35 +1,57 @@
 *** Settings ***
 Resource            ../libs/public-common.robot
-
-Force Tags          GeneralPublic    Local    Dev    Test    Preprod
+Force Tags          GeneralPublic    Prod
 
 Suite Setup         user opens the browser
 Suite Teardown      user closes the browser
 
 *** Test Cases ***
 Navigate to /methodology page
+    [Tags]    Local
     user navigates to public methodologies page
     user waits until page contains
     ...    Browse to find out about the methodology behind specific education statistics and data and how and why they're collected and published.
     ...    10
 
 Validate page contents
-    [Tags]    NotAgainstDev    NotAgainstPreProd    NotAgainstProd
+    [Tags]    Local    NotAgainstProd
     user waits until page contains accordion section    Pupils and schools
     user opens accordion section    Pupils and schools
+
     user opens details dropdown    Exclusions
     user checks page contains methodology link
     ...    Exclusions
     ...    Permanent and fixed-period exclusions in England
     ...    Pupil exclusion statistics: methodology
     ...    %{PUBLIC_URL}/methodology/permanent-and-fixed-period-exclusions-in-england
+
     user opens details dropdown    Pupil absence
     user checks page contains methodology link
     ...    Pupil absence
     ...    Pupil absence in schools in England
     ...    Pupil absence statistics: methodology
     ...    %{PUBLIC_URL}/methodology/pupil-absence-in-schools-in-england
 
+Validate accordion sections exist
+    user checks url contains    methodology
+    user waits until page contains accordion section    Children's social care
+    user waits until page contains accordion section    COVID-19
+    user waits until page contains accordion section    Destination of pupils and students
+    user waits until page contains accordion section    Early years
+    user waits until page contains accordion section    Finance and funding
+    user waits until page contains accordion section    Further education
+    user waits until page contains accordion section    Higher education
+    user waits until page contains accordion section    Pupils and schools
+    user waits until page contains accordion section    School and college outcomes and performance
+    user waits until page contains accordion section    Teachers and school workforce
+    user waits until page contains accordion section    UK education and training statistics
+
+Validate bootstraped methodologies exist
```

Comment:
You still need to update the name of this test case, since it isn't checking a bootstrapped methodology.

I don't think we have any bootstrapped methodology at the moment, and since this keyword is looking for Pupils and schools in `Validate page contents` which runs locally, I think this test case can just be removed.